index.html

```diff
@@ -7,7 +7,17 @@
 </head>
 
 <body>
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
+    <br/>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
+    <form action="/action_page.php">
+        Email: <input type="text" name="Email" value="">
+        <input type="submit" value="Submit">
+    </form>
+    <br/>
+    <br/>
+    <i>Twitter</i><i>FaceBook</i><i>Instagram</i>
```

Comment:
These should be font awesome icons, as per the wireframes.

Also, the need to link to their social media platforms